virtesk-tc-connectspice/conftest.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+# vim: autoindent expandtab tabstop=4 sw=4 sts=4 filetype=python
+
+# Copyright (c) 2013-2016, Adfinis SyGroup AG
+#
+# This file is part of Virtesk VDI.
+#
+# Virtesk VDI is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Virtesk VDI is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Virtesk VDI.  If not, see <http://www.gnu.org/licenses/>.
+
+# System Imports
+import pytest
+
+MOCK_DATA_PATH = "virtesk-tc-connectspice/test_mock_data/"
```

Comment:
This path is relative to where pytest is started... better be absolute...

you can get dir of conftest.py like this:
import os.path

_mock_data_path = os.path.dirname(os.path.realpath(__file__))